zebrad/src/components/mempool/error.rs

```diff
@@ -0,0 +1,22 @@
+//! Errors that can occur when adding transactions to the mempool or querying it.
+
+use thiserror::Error;
+
+#[derive(Error, Copy, Clone, Debug, PartialEq)]
```

Comment:
Oops, this was my mistake.

`Clone` should work after PR #2664 merges, and `main` is merged into this PR.

Or feel free to revert the entire change, and do it in a later PR.

```suggestion
#[derive(Error, Clone, Debug, PartialEq)]
```